packages/cli/src/commands/project/list.ts

```diff
@@ -48,33 +55,38 @@ export default async function list(
   const elapsed = ms(Date.now() - start);
 
   output.log(
-    `${list.length > 0 ? 'Projects' : 'No projects'} found under ${chalk.bold(
-      contextName
-    )} ${chalk.gray(`[${elapsed}]`)}`
+    `${
+      projectList.length > 0 ? 'Projects' : 'No projects'
+    } found under ${chalk.bold(contextName)} ${chalk.gray(`[${elapsed}]`)}`
   );
 
-  if (list.length > 0) {
-    const cur = Date.now();
-    const header = [['', 'name', 'updated'].map(title => chalk.dim(title))];
-
-    const out = table(
-      header.concat(
-        list.map(secret => [
-          '',
-          chalk.bold(secret.name),
-          chalk.gray(`${ms(cur - secret.updatedAt)} ago`),
-        ])
-      ),
+  if (projectList.length > 0) {
+    const tablePrint = table(
+      [
+        ['Project Name', 'Manage', 'Status', 'Updated'].map(header =>
+          chalk.bold(chalk.cyan(header))
+        ),
+        ...projectList
+          .map(project => [
+            [
+              project.name,
+              chalk.bold(getInspectUrl(contextName, project.name)),
+              stateString(project.latestDeployments[0]?.readyState),
+              chalk.gray(ms(Date.now() - project.updatedAt)),
+            ],
+          ])
+          // flatten since the previous step returns a nested
+          // array of the deployment and (optionally) its instances
+          .flat(),
```

Comment:
could you just remove the nested array now? It looks like its unnecessary

Reply:
The nested array is necessary—it's creating an array containing the pieces of data to appear under the header for each project, generating:

```
['project name', 'manage', 'updated'],
['name 1', 'vercel.com/user/name-1', '35s'],
['name 2', 'vercel.com/user/name-2', '1m']
// ...etc
```